test/fixtures/ensembles.py

```diff
@@ -81,7 +87,14 @@ def _make(
         assert not (X is None) ^ (y is None)
 
         if not models:
-            models = [MyDummyRegressor(config=1, random_state=seed) for _ in range(5)]
+            models = [
+                MyDummyRegressor(
+                    feat_type={i: "numerical" for i in range(4)},
```

Comment:
I feel like this is going to fail if X data doesn't match this description. Could we make `feat_types` an optional argument that defaults to None and just does `{i: "numerical" for i in range(X.shape[1])`